<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="accLbl">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Accuracy</string>
    </property>
   </widget>
   <widget class="QLabel" name="accValue">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>80.02%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>1099</width>
      <height>536</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1078</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>508</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="PlotPlaceholder" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(207, 207, 207);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>101</y>
      <width>237</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="fsLbl">
       <property name="text">
        <string>Sampling Frequency</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fsTextbox">
       <property name="text">
        <string>250</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="plotupdateLbl">
       <property name="text">
        <string>Plot Update (msec)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="plotupdateTextbox">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="nchannelLbl">
       <property name="text">
        <string>No. of channels</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="nchannelsTextbox">
       <property name="text">
        <string>30</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="nclassesLbl">
       <property name="text">
        <string>No. of Classes</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="nclassesTextbox">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="errorLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(245, 245, 245);
border: 1px solid;
border-color: rgb(185, 185, 185);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ERROR LOGS</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>210</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="PredictedLabelsLayout">
     <item>
      <widget class="QLabel" name="predicted_0">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(245, 245, 245);
border: 2px solid;
border-color: rgb(185, 185, 185);</string>
       </property>
       <property name="text">
        <string>Left</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="predicted_1">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(245, 245, 245);
border: 2px solid;
border-color: rgb(185, 185, 185);</string>
       </property>
       <property name="text">
        <string>Right</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="predicted_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(245, 245, 245);
border: 2px solid;
border-color: rgb(185, 185, 185);</string>
       </property>
       <property name="text">
        <string>Up</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="predicted_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(245, 245, 245);
border: 2px solid;
border-color: rgb(185, 185, 185);</string>
       </property>
       <property name="text">
        <string>Down</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>160</y>
      <width>71</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>True Class</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>210</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Predicted Class</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>263</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="filepathLbl">
       <property name="text">
        <string>File Path</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="filepathTextbox">
       <property name="text">
        <string>250</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="setfileBtn">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>160</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="TrueLabelsLayout">
     <item>
      <widget class="QLabel" name="true_0">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(245, 245, 245);
border: 2px solid;
border-color: rgb(185, 185, 185);</string>
       </property>
       <property name="text">
        <string>Left</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="true_1">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(245, 245, 245);
border: 2px solid;
border-color: rgb(185, 185, 185);</string>
       </property>
       <property name="text">
        <string>Right</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="true_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(245, 245, 245);
border: 2px solid;
border-color: rgb(185, 185, 185);</string>
       </property>
       <property name="text">
        <string>Up</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="true_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(245, 245, 245);
border: 2px solid;
border-color: rgb(185, 185, 185);</string>
       </property>
       <property name="text">
        <string>Down</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
